src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/RejectedFilesHandler.java

```diff
@@ -88,7 +88,7 @@ private void handleEnvelope(
                 logger.info("Rejected file successfully handled. " + loggingContext);
             }
         } catch (Exception exc) {
-            logger.error("Error handling rejected file. " + loggingContext);
+            logger.error("Error handling rejected file. " + loggingContext, exc);
```

Comment:
just seeing this. in logging statements we should use it's own string interpolation. e.g. `log.level("some string {}", param, exception);`

Reply:
fixed

Reply:
and the other 2 instances in this method?

Reply:
I'll change that separately.